Source/Core/VideoCommon/FPSCounter.h

```diff
@@ -11,20 +11,26 @@
 class FPSCounter
 {
 public:
-  // Initializes the FPS counter.
   FPSCounter();
+  ~FPSCounter();
 
   // Called when a frame is rendered (updated every second).
   void Update();
 
   float GetFPS() const { return m_fps; }
+  double GetDeltaTime() { return m_time_diff_secs; }
```

Comment:
`double GetDeltaTime() const { return m_time_diff_secs; }`

Reply:
OMG not sure how I missed that.